cluster/gce/gci/configure.sh

```diff
@@ -296,8 +296,8 @@ function install-node-problem-detector {
   fi
 
   echo "Downloading ${npd_tar}."
-  local -r npd_release_path="${NODE_PROBLEM_DETECTOR_RELEASE_PATH:-https://storage.googleapis.com/kubernetes-release}"
-  download-or-bust "${npd_hash}" "${npd_release_path}/node-problem-detector/${npd_tar}"
+  local -r npd_release_path="${NODE_PROBLEM_DETECTOR_RELEASE_PATH:-"https://github.com/kubernetes/node-problem-detector/releases/download/${npd_version}/node-problem-detector"}"
+  download-or-bust "${npd_hash}" "${npd_release_path}/${npd_tar}"
```

Comment:
we should manually check job logs
/hold
since CI clearly doesn't catch if this has a typo

Reply:
pull-kubernetes-node-e2e-containerd-standalone-mode does not collect node installation log and npd log. We may need to update https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/services/logs.go#L65 to get logs...

Reply:
Thanks for the pointer, yeah I don't see anything relevant in the artifacts, we'll have to patch in some confirmation

Reply:
Figured it out that we have https://testgrid.k8s.io/sig-node-node-problem-detector#ci-npd-e2e-kubernetes-gce-gci to cover the standalone mode. That CI job is failing now due to some other reason. I will fix that and prepare another PR for you to review to fix this. @BenTheElder